uber/config.py

```diff
@@ -102,20 +102,16 @@ def get_oneday_price(self, dt):
     def get_presold_oneday_price(self, badge_type):
         return self.BADGE_PRICES['single_day'].get(self.BADGES[badge_type], self.DEFAULT_SINGLE_DAY)
 
-    def get_attendee_price(self, dt):
+    def get_attendee_price(self, dt=None):
         price = self.INITIAL_ATTENDEE
         if self.PRICE_BUMPS_ENABLED:
 
-            if c.HARDCORE_OPTIMIZATIONS_ENABLED:
-                # WARNING: EXTREMELY AGGRESSIVE. Don't run the DB query that gets # of badges sold in order
-                # to lighten the server load / blocking time on the DB. THIS ****BREAKS**** BADGE PRICE INCREASES
-                # THAT ARE BASED ON THE NUMBER OF TICKETS SOLD.  Only turn this on if you know EXACTLY what
-                # you are doing, your server is having its face melted off, and you have no other options.
-                # YOU HAVE BEEN WARNED!!!! -Dom
-                badges_sold = 0
-            else:
+            if not dt or c.HARDCORE_OPTIMIZATIONS_ENABLED:
```

Comment:
This now ignores badges_sold any time we pass a datetime variable in. I've changed the calls to never pass it in unless the attendee has a `registered` date (e.g., is not new). The warning about hardcore_optimizations_enabled is already in configspec.ini so I ended up deleting it from here because it was confusing otherwise.